### PR TITLE
added tagging support (ffmpeg metadata - artist,title,album)

### PR DESCRIPTION
**Pull request by dominic-miglar**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [ ] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [ ] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [X] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Explanation of your *pull request* in arbitrary form goes here. Please make sure the description explains the purpose and effect of your *pull request* and is worded well enough to be understood. Provide as much context and examples as possible.

### CODE REVIEW

File: youtube_dl/postprocessor/ffmpeg.py [hunk 1]
```diff
@@ -290,6 +290,16 @@ def run(self, information):
 
         try:
             self._downloader.to_screen('[ffmpeg] Destination: ' + new_path)
+
+            if information.get('artist') is not None:
+                more_opts += ['-metadata', 'artist={}'.format(information['artist'])]
```
Comment (dstftw) [inline youtube_dl/postprocessor/ffmpeg.py]: `{}` won't work in python 2.6.

File: youtube_dl/postprocessor/ffmpeg.py [hunk 2]
```diff
@@ -290,6 +290,16 @@ def run(self, information):
...
+
+            if information.get('title') is not None:
+                more_opts += ['-metadata', 'title={}'.format(information['title'])]
+
+            if information.get('album') is not None:
+                more_opts += ['-metadata', 'album={}'.format(information['album'])]
+
```
Comment (dstftw) [inline youtube_dl/postprocessor/ffmpeg.py]: What's the point of this at all? It's already implemented in `FFmpegMetadataPP`.

**Comment by dominic-miglar** [comment]
When specifying mp3 as Output Type that information won't be set

**Comment by yan12125** [comment]
> When specifying mp3 as Output Type that information won't be set

It's a bug, see #5594

**Comment by dominic-miglar** [comment]
Oh okay, did not see that. Feel free to close that PR.

**Pull request closed without merging**